docs/learn/tokenomics2/Inflation.md

```diff
@@ -2,33 +2,166 @@
+dApp owners also earn rewards, based on how well their dApp is performing in dApp staking, at the end of each **era**.
 
-The rewards for dApp staking, both for users and dApps, will remain unchanged in absolute terms. No immediate modifications will be implemented in this area. Future updates, including the introduction of dApp staking v3, will occur in the third phase of Tokenomics 2.0.
+### Eras
 
-We use the blockReward distribution to ensure those three requirements (and calculate them from the reward per block):
+Each **period** is divided up into multiple **eras**.
+**Era** is the core time unit in dApp staking, and its length is measured in the number of blocks.
 
-|                  | Percentage | Reward ASTR |
-|------------------|------------|-------------|
-| Block reward     | 100%       | 231.20      |
-| Treasury         | 4.78%      | 11.06       |
-| Collators        | 3.06%      | 7.07        |
-| dApp reward      | 17.27%     | 39.93       |
-| Base Staker      | 23.04%     | 53.27       |
-| Ajustable Staker | 51.84%     | 119.85      |
\ No newline at end of file
+`BuildAndEarn` subperiod consists of one or more **eras**. Each era has a fixed length.
+`Voting` subperiod always consists of exactly **one era** but its length can be longer than a standard era length.
+
+### Example
+
+* Block is produced every 12 seconds
+* **Era length** is 7200 blocks which equals 24 hours (1 day)
+* `Voting` subperiod length is 10 eras
+* `Build&Earn` subperiod length is 81 eras
+* Cycle length is **4 periods**
+
+With such configuration, we'd end up with a cycle lasting 364 days (roughly a year), and each period taking around 3 months to complete.
+
+## Recalculation
+
+When a new cycle begins, soft-inflation cap is recalculated, and according to the calculated value, rewards for all network actors are adjusted.
+
+The reason why it's a _soft-cap_, instead of a _hard-cap_ is how staker & dApp rewards are distributed.
+Rewards are only minted when they are claimed, and it is possible that at the time of reward recalculation, some rewards remain unclaimed.
+As a result, theoretically, it's possible for the cycle inflation to exceed the _soft-cap_ even though in practice it will be highly unlikely
+due to various _burn_ mechanisms.
+
+## Inflation Distribution
+
+Based on the calculated _soft-cap_, rewards for all network participants are adjusted.
+
+### Collators
+
+_Collators_ get a fixed amount of the cycle's _soft-capped_ inflation.
+This amount is equally divided by the number of blocks in the cycle.
+
+$collator\_reward\_per\_block = \frac{total\_collator\_cycle\_reward}{blocks\_per\_cycle}$
+
+### Treasury
+
+Similar to the _collators_, treasury gets a fixed amount of the cycle's _soft-capped_ inflation.
+To avoid it being minted all at once, the amount is equaly divided by the number of blocks in the cycle.
```

Comment:
Why is it important to underline "avoid all being minted at once"? 

Why just not write (simplify):
Similar to the _collators_, treasury gets a fixed amount of the cycle's _soft-capped_ inflation distributed in equal amounts throughout all the blocks in the cycle.

Reply:
We can omit that.